packages/wrangler/src/utils/ci-check.ts

```diff
@@ -0,0 +1,34 @@
+import ci from "ci-info";
+import { readConfig } from "../config";
+
+/**
+ * Inside a CI/CD environment, you want to check if all the required variables are in scope
+ * and if not, throw a helpful error message with missing variables and documentation link.
+ */
+export function ciCheck() {
+  if (ci.isCI) {
+    const config = readConfig(undefined, {});
+
+    if (
+      !process.env.CLOUDFLARE_API_TOKEN &&
```

Comment:
we should also check whether a token exists in `~/.wrangler/default`

Reply:
rather, whether Localstate.token got populated

Reply:
Ok, I can do that.

Reply:
Done.